src/sentry/eventstore/base.py

```diff
@@ -213,21 +212,10 @@ def get_event_by_id(self, project_id, event_id, group_id=None):
         """
         raise NotImplementedError
 
-    def get_next_event_id(self, event, snuba_filter):
+    def get_prev_next_event_ids(self, event, snuba_filter):
```

Comment:
Nit: Not sure if better, but `get_adjacent_events_ids`?

Reply:
This is much better hah - I did _not_ like having to write prev_next so much. Will change